File: Scrabble/Scrabble/Scripts/custom/Member.ts
function genericMemberBegin(): void {
	$(".loading").fadeIn();
	$("#genericMemberResults").html("Loading...");
}

function genericMemberFailure(): void {
	$(".loading").fadeOut();
	$("#genericMemberResults").fadeIn();
	$("#genericMemberResults").html("An Error Occured");	
}

function createMemberSuccess(data): void {
	$(".loading").fadeOut();
	$("#genericMemberResults").fadeIn();
	$("#genericMemberResults").html(`Create Member Result: ${data}`);
	if (data === "True") {
		$("#FirstName").val("");
		$("#LastName").val("");
		$("#TelephoneNumber").val("");
		$("#EmailAddress").val("");
		$("#Address1").val("");
		$("#Address2").val("");
		$("#City").val("");
		$("#Region").val("");
		$("#PostCode").val("");
	}
}

function updateMemberSuccess(data): void {
	$(".loading").fadeOut();
	$("#genericMemberResults").fadeIn();
	$("#genericMemberResults").html(`Update Member Result: ${data}`);
}

File: Scrabble/DataLayer/Models/Member.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataLayer.Models
{
	public class Member
	{
		public int Id { get; set; }
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public string TelephoneNumber { get; set; }
		public string EmailAddress { get; set; }
		public string Address1 { get; set; }
		public string Address2 { get; set; }
		public string City { get; set; }
		public string Region { get; set; }
		public string PostCode { get; set; }
		public DateTime DateJoined { get; set; }

	}
}
